app/helpers/quadicon_helper.rb

```diff
@@ -645,6 +699,15 @@ def render_single_quad_quadicon(item, options)
     output.collect(&:html_safe).join('').html_safe
   end
 
+  def img_for_health_state(item)
+    case item.health_state
+    when "Valid"    then "svg/healthstate-normal.svg"
+    when "Critical" then "svg/healthstate-critical.svg"
+    when "None"     then "svg/healthstate-unknown.svg"
```

Comment:
Again hard to tell without real data, but when `health_state` is nil, you get an ugly base-single in the fourth quadrant, may want to make `"None"` the default case.